libssh build failing

https://oss-fuzz-build-logs.storage.googleapis.com/index.html

@cryptomilk - can you please take a look.

@cryptomilk - i see still failures even after your fix and my minor fix.

Can you please send a pull request with fix for this

```
+ clang++ -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fsanitize=address -fsanitize-address-use-after-scope -fsanitize=fuzzer-no-link -stdlib=libc++ -std=c++11 -Iinclude/ /src/libssh/tests/fuzz/ssh_server_fuzzer.cpp -o /out/libssh_server_fuzzer -lFuzzingEngine ./src/libssh.a -Wl,-Bstatic -lcrypto -lz -Wl,-Bdynamic
/src/libssh/tests/fuzz/ssh_server_fuzzer.cpp:27:10: fatal error: 
      'libssh/libssh.h' file not found
#include <libssh/libssh.h>
         ^~~~~~~~~~~~~~~~~
1 error generated.
fuzzers build failed.
```
Please verify build works using python infra/helper.py build_fuzzers libssh

@cryptomilk - now running into this error. would really appreciate if you can please verify locally with build_fuzzers command. we are a small team with limited debugging cycles :(

```
[100%] Built target libsshpp
+ clang++ -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fsanitize=address -fsanitize-address-use-after-scope -fsanitize=fuzzer-no-link -stdlib=libc++ -std=c++11 -Iinclude/ /src/libssh/tests/fuzz/ssh_server_fuzzer.cpp -o /out/libssh_server_fuzzer -lFuzzingEngine ./src/libssh.a -Wl,-Bstatic -lcrypto -lz -Wl,-Bdynamic
/src/libssh/tests/fuzz/ssh_server_fuzzer.cpp:27:10: fatal error: 
      'libssh/libssh.h' file not found
#include <libssh/libssh.h>
         ^~~~~~~~~~~~~~~~~
1 error generated.
fuzzers build failed.
```

--- a/projects/libssh/build.sh
+++ b/projects/libssh/build.sh
@@ -22,7 +22,7 @@ cmake -DCMAKE_C_COMPILER="$CC" -DCMAKE_CXX_COMPILER="$CXX" \
     -DWITH_STATIC_LIB=ON ..
 make "-j$(nproc)"
 
-$CXX $CXXFLAGS -std=c++11 -Iinclude/ \
+$CXX $CXXFLAGS -std=c++11 -I$SRC/libssh/include/ \
     "$SRC/libssh/tests/fuzz/ssh_server_fuzzer.cpp" \
     -o "$OUT/libssh_server_fuzzer" \
     -lFuzzingEngine ./src/libssh.a -Wl,-Bstatic -lcrypto -lz -Wl,-Bdynamic

